_source/user-guide/infrastructure-monitoring/annotations.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: article
+title: Annotations
+permalink: /user-guide/infrastructure-monitoring/annotations/
+flags:
+  logzio-plan: community
+tags:
+  - metrics
+contributors:
+  - shalper
+  - daniel-tk
+---
+
+Annotations can mark events on your Metrics dashboards based on data from a logging account. For example, an annotation can indicate when a particular error log or configuration change occured based on logs written to your main logging account.
+
+Annotations offer another way to correlate your metrics with your logs, to help you figure out _why_ things are happening. Annotations are similar to using [Markers]({{site.baseurl}}/user-guide/insights/markers.html) in your logging [Insights]({{site.baseurl}}/user-guide/insights/exploring-insights.html).
+
+
+
+#### Configure an annotation
+
+<div class="tasklist">
+
+##### Pick your dashboard
+
+Before you begin, you'll need to make sure you have permissions to edit the dashboard.
+
+If you are using any of the pre-configured dashboards provided by Logz.io, you'll first need to select the option to make it editable. (You can also click the gear **<i class="li li-gear"></i>** in the top menu, then select **Save as > Save**).
+
+##### Add a new annotation
+
+Go to the dashboard's settings:
+
+* Click the gear **<i class="li li-gear"></i>** in the dashboard's top menu (upper right corner).
+* Select **Annotations** > **New**.
+* The annotation form will appear. Fill in the form, starting with the **General** section.
+  1. Give a short **Name**. The name should describe the significance of the event.
+  2. Select a logging account as the **Data source**. This is the relevant Kibana account.
+
+<video autoplay loop>
+  <source src="https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana-videos/grafana7-annotations.mp4" type="video/mp4" />
+</video>
+
+##### Decide which logs should trigger the annotation
+
+Type in a _Kibana_ query in Lucene syntax.
+
+The query determines which logs will be marked as events in your dashboard.
+It's best if you double-check your query results in [Kibana Discover](/user-guide/kibana/) to make sure it is specific enough.
+
+Note that the annoation query does not support Grafana variables.
```

Comment:
I think that's not true (@Noa-Levi  correct me if i'm wrong).
You *can* use Grafana variables, on the one condition that the value of the variable actually appears in the target log
Example query: logtype:configuration AND pod.name:$pod (assuming fields logtype amd pod.name exist in the log).

Reply:
@daniel-tk Can you help me clarify this point? I'm assuming there are also syntax issues to note. See: https://grafana.com/docs/grafana/latest/features/datasources/elasticsearch/#using-variables-in-queries